packages/iTwinUI-react/src/core/ComboBox/ComboBoxInput.tsx

```diff
@@ -52,13 +54,25 @@ export const ComboBoxInput = React.forwardRef(
             }
 
             if (focusedIndexRef.current === -1) {
+              const currentElement = menuRef.current?.querySelector(
+                '[data-iui-index]',
+              );
               return dispatch([
                 'focus',
-                Object.values(optionsExtraInfoRef.current)?.[0]
-                  .__originalIndex ?? -1,
+                Number(currentElement?.getAttribute('data-iui-index') ?? 0),
               ]);
             }
 
+            // If virtualization is enabled, dont let round scrolling
+            if (
+              enableVirtualization &&
```

Comment:
Can't you dispatch focus action with 0? Then virtualized menu would set correct `scrollToIndex`

Reply:
amm.. what? I jsut dont want to move at all, so keeping current element focused.
Or I misunderstood your comment